website/blog/2025-08-25-podman-desktop-availability-on-rhel10.md

```diff
@@ -0,0 +1,85 @@
+
+- A curated catalog of open-source models available for downloading.
+- A playground environment for experimenting with models and customizing prompts.
+- A recipe catalog with sample AI use cases.
+- An integration with LlamaStack from your local environment, run containerized distributions, and access LlamaStack APIs.
+- Code snippets for various languages and frameworks, simplifying the infusion of AI into your applications.
+- An inference server to serve models locally with an OpenAI-compatible API.
+
+This enables developers to easily get started with generative AI and integrate models into their applications.
+
+## Availability and installation on RHEL 10
+
+With Podman Desktop availability in the RHEL extensions repository channel, you can easily download and use it. This repository is a curated collection of modern developer tools, open-source libraries, and specialized utilities, ensuring reliable, up-to-date, and Red Hat-signed software.
+For installation, you need a RHEL 10 machine that is registered with the subscription manager using either your [account details](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/configuring_basic_system_settings/assembly_registering-the-system-and-managing-subscriptions_configuring-basic-system-settings#registering-a-system-by-using-the-command-line_assembly_registering-the-system-and-managing-subscriptions) or an [activation key](https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_rhel_system_registration/basic-reg-rhel-cli#proc-reg-rhel-rhc-act-key_).
+
+The installation and verification procedure includes:
+
+1. Open a terminal, and enable the RHEL extensions repository:
+
+   ```sh
+   $ sudo subscription-manager repos --enable rhel-10-for-$(arch)-extensions-rpms
+   ```
+
+1. Enter your password when prompted.
+1. Install Podman Desktop:
+   ```sh
+   $ sudo dnf install podman-desktop
+   ```
+1. Enter `y` to confirm that the installed size is okay.
+1. Enter `y` to import the GPG key and complete the installation.
+1. Enter Podman Desktop in the search box at the top of your home screen, and click the application to open it.
+1. Follow the prompts to complete a quick onboarding process with the application.
+1. Run basic tasks, such as:
+   - [Start a container](/docs/containers/starting-a-container)
+   - [Create a Kubernetes cluster](/docs/kubernetes/creating-a-kube-cluster)
+
+### Important note on support
+
+Podman Desktop is an open-source community project. While Podman itself has supported options on RHEL, the support model for Podman Desktop for Red Hat customers falls under _Cooperative Community Support_. This means it is intended for non-production support and only offers benefits from community contributions, though with no or limited Service Level Agreements (SLAs). Entitled customers can use this level of support to raise non-production queries and open tickets for more complex issues.
```

Comment:
The explanation is very precise and crucial for our RHEL customers to understand. Thanks for including it!